modules/core/middleware/middleware.go

```diff
@@ -0,0 +1,128 @@
+package middleware
```

Comment:
I'm questioning if we should call this middleware. Seems like it would be very confusing given our notion of middleware. We should just put this in the `core/keeper`

Reply:
It's the [name that SDK has adopted for what were previously known ante handlers](https://github.com/cosmos/cosmos-sdk/pull/10028).

Reply:
Yea, `core/keeper/sdk_middleware.go` sounds good to me